OCPBUGS-38402: Updated the Example policy configurations for differen… (#80393)
openshift/openshift-docs: modules/virt-nmstate-example-policy-configurations.adoc

// Module included in the following assemblies:
//
// * networking/k8s_nmstate/k8s-nmstate-updating-node-network-config.adoc

:_mod-docs-content-type: CONCEPT

> [id="virt-nmstate-example-policy-configurations"]

Reviewer: ```suggestion
[id="virt-nmstate-example-policy-configurations_{context}"]
```
Each module should have a module anchor with {context} that must be lowercase and must match the module’s file name. See https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#module-file-metadata.

Author: Correct. My apologies. My own fault for copying and pasting certain file metadata.

Author: :heavy_check_mark:

= Example policy configurations for different interfaces

> Before reading the different example `NodeNetworkConfigurationPolicy` (NNCP) manifest configurations, consider the following factors when applying a policy so that your cluster runs at its best performance conditions:

Reviewer: ```suggestion
Before you read the different example `NodeNetworkConfigurationPolicy` (NNCP) manifest configurations, consider the following factors when you apply a policy so that your cluster runs at its best performance conditions:
```

Author: :heavy_check_mark:

> * When you need to apply a policy to more than one node, create a `NodeNetworkConfigurationPolicy` manifest for each target node. The Kubernetes NMState Operator applies the policy to each node with an NNCP in an unspecified order. Scoping a policy with this approach reduces the length of time for policy application but risks a cluster-wide outage if an error exists with the cluster's configuration. You can avoid this type of error by initially applying NNCP to some nodes and after confirming they are configured correctly, proceed with applying the policy to the remaining nodes.

Reviewer: ```suggestion
* When you need to apply a policy to more than one node, create a `NodeNetworkConfigurationPolicy` manifest for each target node. The Kubernetes NMState Operator applies the policy to each node with an NNCP in an unspecified order. Scoping a policy with this approach reduces the length of time for policy application but risks a cluster-wide outage if an error is in the cluster's configuration. To avoid this type of error, initially apply NNCP to some nodes, and after you confirm they are configured correctly, proceed with applying the policy to the remaining nodes.
```

Author: :heavy_check_mark:
